src/assets/scss/components/_accordion.scss

```diff
@@ -0,0 +1,164 @@
+/**
+* ----------------------------------------------
+* Demo styles
```

Comment:
🐘 Should we file off the serial numbers before merging this?

Reply:
LoL I thought it was good to leave a hint about where it's come from